README.md

```diff
@@ -16,13 +17,15 @@ contribute to such a project.
 
 I did look a lot at the code in the TODO plugin as a basis for my own implementation. I do not think that I have violated any licenses doing so since I have never just copied code from the plugin. If there are any places that can be problematic regarding any licenses, please contact me so that I can resolve the issues. I want to contribute to the Open Source Community, but it is not my intention to step on any toes in the process.
 
-## 2. Pre-Build Plugins
-I do create pre-build releases for each version of the plugin that I tag. The latest release can be obtained from the "Releases" page. Read the README.txt file associated with the release for information on how to install the plugin into the relevant Release version of Qt Creator.
+## 2. Pre-Built Plugins
+
+I do create pre-built releases for each version of the plugin that I tag. The latest release can be obtained from the ["Releases"](releases/) page. Read the README.txt file associated with the release for information on how to install the plugin into the relevant Release version of Qt Creator.

```

Comment:
The "Releases" link does not open to the correct location:
it resolves to: https://github.com/Code7R/SpellChecker-Plugin/blob/bugfix/README-spelling/releases
instead of: https://github.com/Code7R/SpellChecker-Plugin/releases
(it might be because this is on a branch?)